src/loaddb/load_driver.cpp

```diff
@@ -71,6 +73,7 @@ namespace cubload
     m_error_handler = error_handler;
     m_scanner = new scanner (m_semantic_helper, *m_error_handler);
     m_is_initialized = true;
+    m_lines_inserted = 0;
```

Comment:
Driver is initialized only once, while inserted lines must be reset on each batch.

Reply:
I think it is actually reset on each batch by `driver.clear ()`.